scripts/tests/test_validate_docstrings.py

```diff
@@ -229,6 +229,27 @@ def good_imports(self):
         """
         pass
 
+    def no_returns(self):
+        """
+        Say hello and have no returns.
+        """
+        print("Hello World!")
+
+    def empty_returns(self):
+        """
+        Say hello and always return None.
+
+        Since this function never returns a value, this
+        docstring doesn't need a return section.
+        """
+        def say_hello():
+            return "Hello World!"
+        say_hello()
+        if True:
+            return
+        else:
+            return None
```

Comment:
can you add a test for a property that returns something (without the Returns section, which is the correct).

Reply:
such a test was already present before this PR:
```
    def return_not_documented(self):
        """
        Lacks section for Returns
        """
        return "Hello world!"
```

Reply:
it is negative test, not like the one being commented here, so it's inside the BadReturns class